src/main/java/me/mrCookieSlime/Slimefun/Objects/SlimefunItem/machines/electric/AutomatedCraftingChamber.java

```diff
@@ -192,40 +198,65 @@ public boolean isSynchronized() {
 			}
 		});
 	}
-	
+
 	protected void tick(Block b) {
 		if (BlockStorage.getLocationInfo(b.getLocation(), "enabled").equals("false")) return;
 		if (ChargableBlock.getCharge(b) < getEnergyConsumption()) return;
-		
+
 		BlockMenu menu = BlockStorage.getInventory(b);
-		
+
+		String input = getSerializedInput(menu, false);
+		testInputAgainstRecipes(b, input);
+	}
+
+	protected void tickOnce(Block b) {
+		if (ChargableBlock.getCharge(b) < getEnergyConsumption()) return;
+
+		BlockMenu menu = BlockStorage.getInventory(b);
+
+		String input = getSerializedInput(menu, true);
+		testInputAgainstRecipes(b, input);
+	}
```

Comment:
So, now...

These methods are almost identical, so I'd suggest merging them and just adding a parameter to the tick() method instead.